app/controllers/AppealsController.scala

```diff
@@ -151,57 +156,63 @@ class AppealsController @Inject()(val appConfig: AppConfig,
-            s" (correlation ID: $correlationId)")
-          val appeal = appealSubmission.appealInformation
-          logger.debug(s"[AppealsController][submitAppeal] Received caseID response: ${responseModel.caseID} from downstream.")
-          val seqOfNotifications = appeal match {
-            case otherAppeal: OtherAppealInformation if otherAppeal.uploadedFiles.isDefined =>
-              appealService.createSDESNotifications(otherAppeal.uploadedFiles, responseModel.caseID)
-            case _ => Seq.empty
-          }
-          if (seqOfNotifications.nonEmpty) {
-            val redactedNotification = seqOfNotifications.map(notification => notification.copy(file = notification.file.copy(location = "HIDDEN")))
-            logger.debug(s"[AppealsController][submitAppeal] Posting SDESNotifications: $redactedNotification to Orchestrator")
-            fileNotificationOrchestratorConnector.postFileNotifications(seqOfNotifications).map {
-              response =>
-                response.status match {
-                  case OK =>
-                    logger.info(s"[AppealsController][submitAppeal] - Received OK from file notification orchestrator for correlation ID: $correlationId")
-                    val submissionResponseModel = AppealSubmissionResponseModel(caseId = Some(responseModel.caseID), status = OK)
-                    submissionResponseModel
-                  case status =>
-                    PagerDutyHelper.logStatusCode("submitAppeal", status)(RECEIVED_4XX_FROM_FILE_NOTIFICATION_ORCHESTRATOR, RECEIVED_5XX_FROM_FILE_NOTIFICATION_ORCHESTRATOR)
-                    logger.error(s"[AppealsController][submitAppeal] Unable to store file notification for user with enrolment: $enrolmentKey penalty $penaltyNumber (correlation ID: $correlationId) - Received unknown response ($status) from file notification orchestrator. Response body: ${response.body}")
-                    auditStorageFailureOfFileNotifications(seqOfNotifications)
-                    returnErrorResponseIfMultiAppeal(isMultiAppeal)(s"Appeal submitted (case ID: ${responseModel.caseID}, correlation ID: $correlationId) but received $status response from file notification orchestrator")(responseModel.caseID)
-                }
-            }.recover {
-              case e => {
-                logger.error(s"[AppealsController][submitAppeal] Unable to store file notification for user with enrolment: $enrolmentKey penalty $penaltyNumber (correlation ID: $correlationId) - An unknown exception occurred when attempting to store file notifications, with error: ${e.getMessage}")
-                auditStorageFailureOfFileNotifications(seqOfNotifications)
-                returnErrorResponseIfMultiAppeal(isMultiAppeal)(s"Appeal submitted (case ID: ${responseModel.caseID}, correlation ID: $correlationId) but failed to store file uploads with unknown error")(responseModel.caseID)
+      case Left(error) =>
```

Comment:
have you changed flat map to left and right here?

Reply:
no, i added right on top of left to distinguish the two cases more